amti/utils/mturk.py

```diff
@@ -30,10 +31,17 @@ def get_mturk_client(env):
     region_name = settings.ENVS[env]['region_name']
     endpoint_url = settings.ENVS[env]['endpoint_url']
 
+    profile = os.getenv('AWS_PROFILE')
+    if profile is None:
+        session = boto3.session.Session()
+    else:
+        logger.debug(f'Creating mturk session with profile_name {profile}')
+        session = boto3.session.Session(profile_name=profile)
```

Comment:
It's probably worth it to log the debug message in either case, maybe something like:

```python
profile = os.getenv('AWS_PROFILE')
if profile is None:
    logger.debug('Creating mturk session using the default profile.')
    session = boto3.session.Session()
else:
    logger.debug(f'Creating mturk session with profile: {profile}.')
    session = boto3.session.Session(profile_name=profile)
```